app-tasks/src/rf/layer_uploads/activities.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from botoflow.decorators import activities, activity
+
+from ..utils.logger import get_logger
+
+logger = get_logger()
+
+
+@activities(schedule_to_start_timeout=60, start_to_close_timeout=60)
+class LayerImageUploadActivities(object):
+    """Activities that handle various aspects of a layer upload"""
+
+    @activity(os.environ.get('SWF_VERSION', '1.0'))
```

Comment:
Would it make sense to make this is a required variable? Thinking about failure situations, if the environment variable is missing it might take us a while to notice if it silently switches to a fallback version, which could potentially cause task leakage across deployment colors, so it might be safer to have things fail loudly if the environment is missing.